features/timeseries/src/main/java/org/opennms/netmgt/timeseries/util/TimeseriesUtils.java

```diff
@@ -47,9 +47,14 @@
  */
 public final class TimeseriesUtils {
 
-    public static final String PREFIX_EXTERNAL_TAG =  "_ext_";
-
-    public static final int WILDCARD_INDEX_NO = 2; // => node level
+    /**
+     * This constant defines at which level in a resource path we start to query for multiple metrics.
+     * To save queries, we want to get all resources under a node.
+     * requisitions have the node at the 4th position, example: snmp/fs/xx/4/
+     * auto detected metrics have the node at 2nd position, example: snmp/4/mib2-tcp
+     * since we can't distinguish we set the level to 4.
+     */
+    public static final int WILDCARD_INDEX_NO = 4;
```

Comment:
I wonder if we can distinguish between the 2 types of resources (provision vs. auto detect).
Maybe check if the second or 4th element is a number? Not sure if that would work?

Reply:
@j-white what is your take on this?